Libs/MRML/DisplayableManager/vtkMRMLSliceIntersectionWidget.cxx

```diff
@@ -318,12 +318,17 @@ bool vtkMRMLSliceIntersectionWidget::CanProcessInteractionEvent(vtkMRMLInteracti
     }
 
   // If we are currently dragging a point then we interact everywhere
+  // (we do not let any other widget take away the focus).
   if (this->WidgetState == WidgetStateMoveCrosshair
     || this->WidgetState == WidgetStateTranslate
     || this->WidgetState == WidgetStateRotateIntersectingSlices
     || this->WidgetState == WidgetStateBlend
     || this->WidgetState == WidgetStateTranslateSlice
-    || this->WidgetState == WidgetStateZoomSlice)
+    || this->WidgetState == WidgetStateZoomSlice
+    || this->WidgetState == WidgetStateTranslateIntersectingSlicesHandle
+    || this->WidgetState == WidgetStateRotateIntersectingSlicesHandle
+    || this->WidgetState == WidgetStateTranslateSingleIntersectingSliceHandle
```

Comment:
Should `WidgetStateTouchGesture` and `WidgetStateOnTranslateSingleIntersectingSliceHandle` be specified as well ? 

If yes, then checking `this->WidgetState < WidgetState_Last` would be more robust ?

Reply:
We only want to unconditionally claim the events if we are in the middle of a drag-and-drop operation. 

`WidgetStateOnTranslateSingleIntersectingSliceHandle` just means that the mouse is over a handle, it should not capture events.

`WidgetStateTouchGesture` is for handling touch events. I think events during touch gesture state are intentionally not captured (to allow processing some touch gestures in this displayable manager, but letting other displayable managers to process some other touch events).